TestsFoundation/TestsFoundation/Extensions/TestTreeExtensions.swift

```diff
@@ -0,0 +1,117 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2020-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+@testable import Core
+
+extension TestTree: CustomStringConvertible {
+    public var description: String { description(indent: "", after: "").joined(separator: "\n") }
+    private func description(indent: String, after: String) -> [String] {
```

Comment:
It was late at night... I was having fun... I may have gone a little overboard here

Reply:
so is this just for `print(tree)` then? I don't see any uses of it.

Reply:
Yeah, and debugging